### PR TITLE
Add EAC steps to plus-addressing-in-exchange-online.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2927.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/recipients-in-exchange-online/plus-addressing-in-exchange-online.md [hunk 1]
```diff
@@ -26,9 +26,15 @@ The original email address must be valid; the `+tag` value that you add is arbit
...
 
-You can't enable plus addressing in the Exchange admin center (EAC); you can only enable it through Exchange Online PowerShell. If your organization's email is routed through Exchange Online to your on-premises servers, mailboxes hosted on-premises will also be able to use plus addresses.  
+If your organization's email is routed through Exchange Online to your on-premises servers, mailboxes hosted on-premises will also be able to use plus addresses.  
 
-## Enable plus addressing in your Exchange Online organization
+## Use the new Exchange admin center to enable plus addressing
+
+1. In the new [EAC](https://admin.exchange.microsoft.com), go to **Settings** \> **Mail flow**.
+
+2. Select **Turn on plus addressing for your organization** and then click **Save**.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/plus-addressing-in-exchange-online.md:35]:
```suggestion
1. In the new [Exchange admin center](https://admin.exchange.microsoft.com), go to **Settings** \> **Mail flow**.

2. Select **Turn on plus addressing for your organization** and then select **Save**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: Exchange/ExchangeOnline/recipients-in-exchange-online/plus-addressing-in-exchange-online.md (hunk 1 above)
Comment (dariomws) [reply Exchange/ExchangeOnline/recipients-in-exchange-online/plus-addressing-in-exchange-online.md:35]: Let's keep "click" as it's a button.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: Exchange/ExchangeOnline/recipients-in-exchange-online/plus-addressing-in-exchange-online.md [hunk 2]
```diff
@@ -26,9 +26,15 @@ The original email address must be valid; the `+tag` value that you add is arbit
...
 
-You can't enable plus addressing in the Exchange admin center (EAC); you can only enable it through Exchange Online PowerShell. If your organization's email is routed through Exchange Online to your on-premises servers, mailboxes hosted on-premises will also be able to use plus addresses.  
+If your organization's email is routed through Exchange Online to your on-premises servers, mailboxes hosted on-premises will also be able to use plus addresses.  
 
-## Enable plus addressing in your Exchange Online organization
+## Use the new Exchange admin center to enable plus addressing
+
+1. In the new [Exchange admin center](https://admin.exchange.microsoft.com), go to **Settings** \> **Mail flow**.
+
+2. Select **Turn on plus addressing for your organization** and then click **Save**.
```
Comment (tiburd) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/plus-addressing-in-exchange-online.md:35]:
```suggestion
2. Select **Turn on plus addressing for your organization**, and then select **Save**.
```

File: Exchange/ExchangeOnline/recipients-in-exchange-online/plus-addressing-in-exchange-online.md (hunk 1 above)
Comment (tiburd) [reply Exchange/ExchangeOnline/recipients-in-exchange-online/plus-addressing-in-exchange-online.md]: It's Microsoft style to use "select" for buttons:
image

**Pull request merged**